README.rst

```diff
@@ -33,6 +33,13 @@ The ``FiscalYear`` class provides an object for storing information about the st
    >>> a.end
    FiscalDateTime(2017, 9, 30, 23, 59, 59)
 
+You can also get the current ``FiscalYear`` with:
+
+.. code-block:: python
+
+   >>> from fiscalyear import *
```

Comment:
I would drop this import line here and below. After the first code block I assume everyone will have fiscalyear imported.